src/StatsTools/reweighting.jl

```diff
@@ -558,7 +558,10 @@ function rayleigh_replica_estimator(
     kwargs...
 )
     df = DataFrame(sim)
-    num_reps = length(filter(startswith("norm"), names(df)))
+    num_spectral = length(Set(
+        [name[1:findlast('_',name)] for name in names(df) if startswith(name,"norm")]
+    ))
+    num_reps = div(length(filter(startswith("norm"), names(df))),num_spectral)
```

Comment:
Hmm this feels a bit clunky and probably isn't backwards-compatible. Maybe we should store `num_spectral` and `num_replicas` in the data frame and use that. If the metadata is not available, we could resort to assuming it's a data frame from and old version of Rimu and use the old version of the code.

Reply:
See the fields `"num_replicas"` and `"num_spectral_states"` in the report metadata!